examples/with-next-iframe/README.md

```diff
@@ -1,11 +1,55 @@
 # SuperTokens Example
 
-## How to use:
+## Project Setup:
+
+Use `npm` to install the project dependencies:
+
+```bash
+npm install
+```
+
+Start ngrok and expose port `3000`
+
+```bash
+ngrok http 3000
+```
+
+## Run the demo app
+
+-   Put the https domain of ngrok in `config/supertokensConfig.js`
+
+-   Build and serve the Next app.
+
+    ```bash
+    npm run dev
+    ```
 
--   Start server by running `npm run build && npm run start`
--   Start ngrok: `ngrok http 3000`
--   Put the https domain of ngrok in config/supertokensConfig.js
 -   In chrome > settings > privacy > cookies > check block third party cookies in incognito
 -   Open chrome incognito on `http://localhost:3000/iframe`
 
-## IMPORTANT: This example currently doesn't work on incognito chrome and safari due to their third party cookie blocking rules. You can fix this by making use of localstorage instead of cookies. Here is a [guide](https://supertokens.io/recipe-redirect?to=/advanced-customizations/examples/localstorage/about) on how to do it.
+### IMPORTANT: This example currently doesn't work on incognito chrome and safari due to their third party cookie blocking rules. You can fix this by making use of localstorage instead of cookies. Here is a [guide](https://supertokens.io/recipe-redirect?to=/advanced-customizations/examples/localstorage/about) on how to do it.
+
+## Providers credentials
```

Comment:
Remove this section from all the readmes as it's there in the code comment anyway.